pilot/pkg/features/pilot.go

```diff
@@ -232,6 +232,19 @@ var (
 		"If enabled, any HTTP services will be blocked on HTTPS port (443). If this is disabled, any "+
 			"HTTP service on port 443 could block all external traffic",
 	).Get()
+
+	EnableDistributionTracking = env.RegisterBoolVar(
+		"PILOT_ENABLE_DISTRIB_TRACKING",
+		false,
```

Comment:
As a part of the design review in the networking WG, we committed to shipping this as disabled in its first version, and enabling it by default in 1.5.

Reply:
No one will test it if you turn it off by default. We use flags to turn features off when things go wrong and quickly ship a patch fix. So, if you keep this as off, then enabling this to on by default is equally dangerous whether you do it in 1.5 or 1.4 or 2.0. It will be the first time when it will see widespread usage and issues.

Reply:
I spoke to @howardjohn as well. Lets turn it on by default please. If there is memory leak before the release, we can turn this off. This way, we can get the istioctl fix into 1.4 as well. cc @esnible and co - do you guys think you can get the istioctl experimental config-status command out before freeze or a day after freeze?